wmn-data.json

```diff
@@ -5137,7 +5137,18 @@
         "m_string" : "<title>Salon24 - Blogi, wiadomości, opinie i komentarze",
         "m_code" : 301,
         "known" : ["matuzalem", "niewiarygodne"],
-        "cat" : "blog"
+        "cat" : "blog"
+       },
+       {
+        "name" : "Scammer.info",
+        "uri_check" : "https://scammer.info/u/{account}.json",
+        "uri_pretty" : "https://scammer.info/u/{account}",
+        "e_code" : 200,
+        "e_string" : "<title> Profile -",
+        "m_string" : "Page Not Found - Scammer Info",

```

Comment:
This string also needs adjusting to work with the JSON response: `\"errors\":[\"The requested URL or resource could not be found.`

Reply:
@WebBreacher ah yes, when I changed to the .json, I forgot to check the E and m strings.
Will run a cURL shortly and amend

Reply:
no worries. all good